### DISCUSSION TITLE
Exact method under dense-brain-T1-3d-mse-32feat model

### DISCUSSION THREAD

**Discussion by moniquisma** [voxelmorph/voxelmorph#524] [Q&A]
Hi all,

I would like to know what is the exact method used for obtaining the model in

https://github.com/voxelmorph/voxelmorph/blob/dev/data/readme.md#models

[dense-brain-T1-3d-mse-32feat](https://surfer.nmr.mgh.harvard.edu/ftp/data/voxelmorph/models/vxm_dense_brain_T1_3D_mse.h5)

What are the loss functions involved? Does the model use the loss regarding dice accuracy?
Is it small deformation or stationary parametrization?
Does the model use the parameters tuned by HyperReg?
How was the model trained? In which datasets?

Best!

**Comment by adalca** [comment]
Good point, we should include this information -- although we're also likely to update these models soon. This was trained with MSE as the image matching loss with hyperparameter of 0.02 (I think). T1 MPRAGE dataset as given in paper. 32 features at each level. I unfortunately don't recall what deformation parametrization (unfortunately), but it is not tuned by HyperMorph (it predates that).

**Reply by moniquisma** [reply]
Dear Adrian,

Thank you for your soon reply. I think the parametrization was the stationary one, the transformations 
look quite diffeomorphic.
Do you remember whether you included information regarding Dice coefficients in the training?

Best regards!

**Comment by adalca** [comment]
no i don't believe we used Dice during training of that model!

**Reply by moniquisma** [reply]
Thank you!!
